resources/js/admin.js

```diff
@@ -95,4 +95,17 @@ $(function() {
             }
         });
     });
+    $('option').mousedown(function(e) {
+        e.preventDefault();
+        var originalScrollTop = $(this).parent().scrollTop();
+        console.log(originalScrollTop);
+        $(this).prop('selected', $(this).prop('selected') ? false : true);

```

Comment:
`$(this).prop('selected') ? false : true` can be written as `!$(this).prop('selected')`.

Reply:
Fixed.